src/modules/StoryPage/components/organisms/Envision/index.tsx

```diff
@@ -0,0 +1,76 @@
+import { Typography } from 'm3ntorship-ui';
+import vision from 'public/images/story-page/vision.png';
+import { ReactElement } from 'react';
+
+import { NextImage } from '@/shared/components/atoms';
+import { SCREENS } from '@/shared/constants';
+import useWindowSize from '@/shared/hooks/windowSize';
+
+const singlePillar = {
+  id: '1',
+  description:
+    'Sed ut perspiciatis unde omnis iste natus error sit voluptatem accusantium doloremque laudantium, totam rem aperiam.',
+};
+
+const pillars: typeof singlePillar[] = Array(4)
+  .fill(singlePillar)
+  .map((pillar: typeof singlePillar, index) => ({
+    ...pillar,
+    id: `${index + 1}`,
+  }));
+
+export default function Envision(): ReactElement {
+  const isMDScreen = useWindowSize(`(min-width: ${SCREENS.MD}px)`);
+  const isLGScreen = useWindowSize(`(min-width: ${SCREENS.LG}px)`);
+
+  return (
+    <section className='flex flex-col gap-x-8 gap-y-12 large:flex-row'>
```

Comment:
we need to increase the vertical gap between the image on the lift and pillars on the right 
```suggestion
    <section className='flex flex-col gap-x-24 gap-y-12 large:flex-row'>
```